### ISSUE TITLE
Wrong activation maps.

### ISSUE THREAD

**Issue by chrisrn** [jazzsaxmafia/Weakly_detector#7]
I am using your technique to obtain activation maps for only one class in a binary classification problem. The network I have trained is inception v3. I cut the last 3 inception modules and then I added a new convolutional layer, a gap layer and a softmax. The input of the new convolutional layer has shape batch_size x 17 x 17 x 768 and the output's shape is batch_size x 17 x 17 x 1024, using 3 x 3 kernel. The precision of the network is high (89%). Do you have idea what might be wrong? I have not changed your technique.
